syncmaster/backend/settings/__init__.py

```diff
@@ -41,6 +35,15 @@ class BackendSettings(SyncmasterSettings):
         SYNCMASTER__SERVER__DEBUG=True
     """
 
+    TZ: str = "UTC"
```

Comment:
Does server use this variable?

Reply:
Same for worker and scheduler

Reply:
scheduler need TZ: 
`self.scheduler = AsyncIOScheduler(timezone=settings.TZ)`

Reply:
It looks like we could move this to SchedulerSettings class, and remove from WorkerAppSettings and SchedulerAppSettings.

BTW, could we instead use local timezone instead of overriding it? Just a question, not a proposal.

Reply:
why not? we could use local timezone (it is by default in `BaseScheduler`)